nifi-nar-bundles/nifi-elasticsearch-bundle/pom.xml

```diff
@@ -39,7 +39,7 @@ language governing permissions and limitations under the License. -->
          Note: the low-level elasticsearch-rest-client remains licensed with Apache 2.0
          (https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/_license.html) even after the move
          of the main Elasticsearch product and elasticsearch-rest-high-level-client to Elastic 2.0/SSPL 1.0 in v7.11.0+ -->
-        <elasticsearch.client.version>7.13.4</elasticsearch.client.version>
+        <elasticsearch.client.version>7.17.10</elasticsearch.client.version>
```

Comment:
The problem with upgrading this version is documented in the comment above within the `pom.xml` - this is likely to break connectivity to any non-Elastic based services that use the Elasticsearch API (e.g. AWS OpenSearch)

What CVE are you attempting to address and does it impact these Elasticsearch low-level REST API or (more likely) the Elasticsearch server at this version, which **is not** used by NiFi?

Reply:
Suggest checking the suppressions (and their justifications) in the OWASP Dependency Check [suppression.xml](https://github.com/apache/nifi/blob/main/nifi-dependency-check-maven/suppressions.xml#L107) - there's are several for Elasticsearch, have you detected a known "false positive" that instead you can suppress/ignore in your own checks?

Reply:
Sure, I'll look into those OWASP first.
Closing this PR for now.